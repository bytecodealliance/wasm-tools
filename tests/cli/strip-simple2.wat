;; RUN: strip % -t

(module
  (func $foo)

  (@custom "foo" "...")
)
