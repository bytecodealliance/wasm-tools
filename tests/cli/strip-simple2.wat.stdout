(module
  (type (;0;) (func))
  (func $foo (;0;) (type 0))
)